src/FantasyFun.API/Models/Team.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace FantasyFun.API.Models
+{
+    public class Team
+    {
+        public long Id { get; set; }
+        public long FifaApiId { get; set; } 
+        public long ApiID { get; set; }
```

Comment:
Maybe better is stick to one convention - Id not ID

Reply:
ok